app/requirements.txt

```diff
@@ -1,9 +1,14 @@
+certifi==2017.7.27.1
+chardet==3.0.4
 click==6.7
-flask==0.12.2
+Flask==0.12.2
+gunicorn==19.7.1
+idna==2.6
 itsdangerous==0.24
-jinja2==2.9.6
-markupsafe==1.0
-werkzeug==0.12.2
-wtforms==2.1
+Jinja2==2.9.6
+MarkupSafe==1.0
 pyflakes==1.6.0
-gunicorn
\ No newline at end of file
+requests==2.18.4
+urllib3==1.22

```

Comment:
adding urllib and requests - we can one of those

Reply:
without requesr urllib wont work